app/chat_endpoint_handlers.py

```
@@ -154,6 +155,17 @@
 
 async def chat_message_internal(chat_id: str, user_id: str, action: Action, background_tasks: BackgroundTasks):
     await prewarm_gamemaster()
+    cursor = sql_connection.cursor()
+    cursor.execute(
+        "SELECT remaining_messages FROM chat_users.users WHERE user_id=?;",
+        [user_id]
+    )
+    remaining_messages = 0
```

Ruff (0.12.2)

Redefinition of unused `remaining_messages` from line 7

(F811)